getting-started/dc2-guide.rst

```diff
@@ -1,165 +1,223 @@
-########################################
-Processing DC2 data with the Gen3 Butler
-########################################
+#########################################
+Processing DC2 data with the AP Pipelines
+#########################################
 
-The data explored in this guide is simulated data, the same kind used for Rubin's Data Preview 0 (DP0).
+A walkthrough for running the Alert Production (AP) pipeline on an example set of image data. The data used in this guide is simulated data, the same kind used for Rubin's Data Preview 0 (DP0).
 
 .. note::
 
-   This guide assumes the user has access to a shared "Gen3" Butler repository containing data from the Dark Energy Science Collaboration (DESC)'s Data Challenge 2 (DC2), most likely on the ``lsstdevl`` machines at NCSA.
+   This guide assumes the user has access to a shared Butler repository containing data from the Dark Energy Science Collaboration (DESC)'s Data Challenge 2 (DC2) via the US Data Facility (USDF).
    This guide further assumes the user has a recently-built version of :py:mod:`lsst.distrib` from the LSST Science Pipelines.
-   The instructions in this guide were verified to work in July 2021 using ``rubin-env 0.6.0`` with the weekly tagged ``w_2021_30``.
-   Finally, this guide assumes the user is interested in running the Alert Production (AP) pipeline on this data using good seeing templates.
 
-What's in the Shared DC2 Butler Repository?
-===========================================
+What does existing DC2 data look like?
+======================================
 
 * The instrument is called ``LSSTCam-imSim``
 * The obs-package is ``obs_lsst``, i.e., :py:mod:`lsst.obs.lsst` (note ``imsim`` subdirectories within)
```

Comment:
Hadn't realized this, I was under the impression that module links would work once the full page built (a version that knows about the whole stack instead of just pipelines_lsst_io), but good point that it's broken on the live version of the page. Will update in a consistent way.